deploy/kcp/apiexport_jvmbuildservice.yaml

```diff
@@ -0,0 +1,27 @@
+# This file is generated from CRDs by ./hack/generate-kcp-api.sh script.
+# Please do not modify!
+
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIExport
+metadata:
+  name: jvmbuildservice
+spec:
+  permissionClaims:
+  - group: ""
+    resource: "secrets"
+  - group: ""
+    resource: "configmaps"
+  - group: "jvmbuildservice.io"
+    identityHash: 2aa7813102d123ead4faa4e5cdf4b9d87b21eb9d9e0424475c9cb3cfd1963e50
+    resource: "artifacdtbuilds"
```

Comment:
typo, and I think that you do not need to have permissionClaim on resources which are in latestResourceSchemas.

Reply:
thanks, both for the typo, and then, yeah, I was uncertain about that particular .... now that I'm actually testing, I'll be sure and try a permutation with these removed and confirm one way or the other.

Reply:
fyi I've made this adjustment, along with the associated change needed apibinding that we discussed with @mmorhun in standup today and was confirmed during the cabal with the infra team

also been able to verify the receipt of events for our CRDs locally